sdk/devcenter/developer-devcenter-rest/package.json

```
@@ -0,0 +1,127 @@
+  },
+  "//sampleConfiguration": {
+    "productName": "Azure DevCenter",
+    "productSlugs": [
+      "azure"
+    ],
+    "disableDocsMs": true,
+    "apiRefLink": "https://learn.microsoft.com/azure/dev-box/"
+  },
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "scripts": {
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "build:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1",
+    "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1",
+    "build:samples": "echo skipped.",
+    "build:test": "tsc -p . && dev-tool run bundle",
+    "build:debug": "tsc -p . && dev-tool run bundle && api-extractor run --local",
+    "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-browser dist-esm test-dist temp types *.tgz *.log",
+    "execute:samples": "echo skipped",
+    "extract-api": "rimraf review && mkdirp ./review && api-extractor run --local",
+    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
+    "generate:client": "autorest --typescript swagger/README.md && npm run format",
+    "integration-test:browser": "dev-tool run test:browser",
+    "integration-test:node": "dev-tool run test:node-js-input -- --timeout 5000000 'dist-esm/test/**/*.spec.js'",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
+    "lint": "eslint package.json api-extractor.json src test --ext .ts",
+    "pack": "npm pack 2>&1",
+    "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser",
+    "test:node": "npm run clean && npm run build:test && npm run unit-test:node",
+    "test": "npm run clean && npm run build:test && npm run unit-test",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser",
+    "unit-test:node": "dev-tool run test:node-ts-input -- --timeout 12000000 --exclude 'test/**/browser/*.spec.ts' 'test/**/*.spec.ts'",
+    "unit-test:browser": "echo skipped",
+    "build": "npm run clean && tsc -p . && dev-tool run bundle && mkdirp ./review && api-extractor run --local"
+  },
+  "sideEffects": false,
+  "autoPublish": false,
+  "dependencies": {
+    "@azure/core-auth": "^1.3.0",
+    "@azure-rest/core-client": "1.0.0-beta.10",
+    "@azure/core-rest-pipeline": "^1.8.0",
+    "@azure/logger": "^1.0.0",
+    "tslib": "^2.2.0",
+    "@azure/core-paging": "^1.2.0",
+    "@azure/core-lro": "^2.2.0"
+  },
+  "devDependencies": {
+    "@microsoft/api-extractor": "7.18.11",
+    "autorest": "latest",
+    "@types/node": "^12.0.0",
```

We met the ci error https://dev.azure.com/azure-sdk/public/_build/results?buildId=1925208&view=logs&j=efd83a73-1d18-5a8b-eda0-914b9a825166&t=7b68dc2f-1868-549c-7c85-81003212da1d&l=386. 

Could you upgrade the node to 14 manually? 
```suggestion
    "@types/node": "^14.0.0",
```

Another way to fix is to use the latest autorest/typescript extension.